requirements/README.md

```diff
@@ -0,0 +1,122 @@
+3. `h5py` - HDF5 for Python - The [h5py package](https://github.com/h5py/h5py) is a Pythonic interface to the HDF5 binary data format. 
+4. `PyTorch` - Download the suitable version of [PyTorch](https://pytorch.org/) based on your computer. The version of `torch` should be `1.12.0+cpu`.
+5. `DQC` - Download the [Differentiable Quantum Chemistry package](https://github.com/diffqc/dqc).
+6. `xitorch` - [xitorch](https://github.com/xitorch/xitorch) is a PyTorch-based library of differentiable functions and functionals that can be widely used in scientific computing applications as well as deep learning.
+7. `pylibxc2` - A thin [wrapper](https://github.com/mfkasim1/pylibxc/) of libxc. 
+
+## env_mac.yml
+This file contains the requirements specific to macOS. The dependencies are as follows:
+1. `hhsuite` - The [HH-suite](https://github.com/soedinglab/hh-suite) is an open-source software package for sensitive protein sequence searching based on the pairwise alignment of hidden Markov models (HMMs).
+2. `vina` - [AutoDock Vina](https://github.com/ccsb-scripps/AutoDock-Vina) is one of the fastest and most widely used open-source docking engines. It can be installed with `pip`.
+
+## env_ubuntu.yml
+This file contains the requirements specific to the Ubuntu distro of the Linux Operating System. The dependency is as follows:
+1. `hhsuite` - The [HH-suite](https://github.com/soedinglab/hh-suite) is an open-source software package for sensitive protein sequence searching based on the pairwise alignment of hidden Markov models (HMMs).
+
+## env_test.yml
+This file contains the requiremetns to perform testing of models and code in the DeepChem library. The dependencies can be installed using `pip` and are as follows:
+1. `flake8` - [flake8](https://github.com/PyCQA/flake8) is a python tool that glues together pycodestyle, pyflakes, and mccabe.
+2. `flaky` - [Flaky](https://github.com/box/flaky) is a plugin for nose or pytest that automatically reruns flaky tests.
+3. `mypy` - [Mypy](https://github.com/python/mypy) is a static type checker for Python.
+4. `pytest` - The [pytest framework](https://github.com/pytest-dev/pytest) makes it easy to write small tests.
+5. `pytest-cov` - It's a coverage [plugin](https://github.com/pytest-dev/pytest-cov) for `pytest`.
+6. `types-pkg_resources` - This is an auto-generated PEP 561 type stub package for pkg\_resources package. It can be used by type-checking tools like mypy, PyCharm, pytype etc. to check code that uses pkg\_resources. Its source is [here](https://github.com/python/typeshed/tree/master/stubs/pkg_resources).
+7. `types-setuptools` - This is a PEP 561 type stub package for the setuptools package. It can be used by type-checking tools like mypy, pyright, pytype, PyCharm, etc. to check code that uses setuptools. Its source is [here](https://github.com/python/typeshed/tree/main/stubs/setuptools).
+8. `yapf` - It is a [formatter](https://github.com/google/yapf) for Python files.
+
+## jax
+### env_jax.cpu.yml
```

Comment:
In this part, probably we don't need such a long description of what each requirements files are about. Can you make it short, something like a one line description for each file, like `env_jax.cpu.yml` - specifies jax installation requirements for cpu.

Following the same for tensorflow and torch will be helpful.